control/main.py

```diff
@@ -0,0 +1,194 @@
+"""Main program for autonomous drone flight and sensor data logging.
```

Comment:
Should probably explain what we're expecting from the GCS (both waypoints, and that the GCS expects unicode strings and the data it expects. Cuz otherwise without looking at the GCS code the data just looks like we randomly decided it.